list_participants.json

```diff
@@ -8,4 +8,12 @@
     "Social media": "twitter: @pierre_bellec",
     "Repository": "https://github.com/mtl-brainhack-school-2018/brainsprite"
   },
-}
+  {  "Marie Caillaud": {
```

Comment:
Could you please remove this bracket ? It's going to break json 😞

Reply:
can you tell me how to remove that?

Reply:
If you make another commit on your copy of the repository, it will update the pull request here ! Let me know if I can provide any more info to help in that 😄